lib/ecto/adapters/mysql.ex

```diff
@@ -186,4 +191,26 @@ defmodule Ecto.Adapters.MySQL do
     raise ArgumentError, "MySQL does not support :read_after_writes in schemas for non-primary keys. " <>
                          "The following fields in #{inspect schema} are tagged as such: #{inspect returning}"
   end
+
+  @doc false
+  def database_info do
+    { output, _ } = System.cmd("mysql", ["--version"])
```

Comment:
Needless spaces in pattern here and below.
Also in few other places in the PR. :bowtie: